app/commands/build_claim_command.rb

```diff
@@ -17,20 +17,30 @@ class BuildClaimCommand < BaseCommand
   attribute :miscellaneous_information, :string
   attribute :employment_details
   attribute :is_unfair_dismissal, :boolean
+  attribute :pdf_template_reference, :string, default: 'et1-v1-en'
 
-  def initialize(*)
-    super
-    self.reference_service = ReferenceService
+  validates :pdf_template_reference, inclusion: { in: ['et1-v1-en', 'et1-v1-cy'] }
+
+  def initialize(*args, reference_service: ReferenceService, allocator_service: UploadedFileAllocatorService.new, **kw_args)
```

Comment:
This change allows the external services to be injected when running tests on this command.  Normal functionality will not have changed as a result of this